src/main/java/org/folio/rest/impl/InvoicesImpl.java

```diff
@@ -174,6 +188,40 @@ public void deleteInvoiceInvoiceLinesById(String id, String lang, Map<String, St
       .exceptionally(t -> handleErrorResponse(asyncResultHandler, invoiceLineHelper, t));
   }
 
+  @Override
+  public void putInvoiceInvoiceLinesFundDistributionsValidate(ValidateFundDistributionsRequest entity,
```

Comment:
By just eyeballing this method, I see no indication that any processing is deferred(async with Future/callbacks). If this method calls out to external parties or the database, it will be good to relinquish the vertx event loop. My concern is that this method would hog the event loop if the duration of this method is not quick. This method is fine as is, if there arent't any long operations contained.

Reply:
well it is just for validating fund amounts, i dont think it would contain any long operations.

Reply:
Sounds good.